deeplake/util/token.py

```diff
@@ -13,3 +14,48 @@ def is_expired_token(creds: dict) -> bool:
     if "expires_at" not in creds:
         return False
     return creds["expires_at"] < time.time()
+
+
+def src_token_and_dest_token_deprecation_warning(src_token, dest_token, token):
+    if src_token is None and dest_token is None and token is None:
+        return src_token, dest_token
+
+    if src_token or dest_token:
+        warn(
+            "`src_token` and `dest_token` are deprecated, better to use just single `token`",
+            FutureWarning,
+            stacklevel=2,
+        )
+
+    if dest_token and src_token and token:
+        warn(
+            "You are using `dest_token` and `src_token` with `token`. Only `token` will be executed",
```

Comment:
executed seems like the wrong word here

Reply:
we should also mention deprecation here and say that these arguments will be removed in a future release.